h2o-hadoop/scripts/docker/cdh/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM nimmis/java:14.04-oracle-8-jdk
+
+ARG VERSION
+ARG PATH_PREFIX='.'
+ARG SPARK_VERSIONS=''
+
+ENV DISTRIBUTION='cdh' VERSION=$VERSION ENTER_BASH=FALSE HADOOP_CONF_DIR='/etc/hadoop/conf.pseudo' MASTER='yarn-client'
+
+RUN echo "# Packages for Cloudera's Distribution for Hadoop, Version 5, on Ubuntu 14.04 amd64\n\
+deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh${VERSION} contrib\n\
+deb-src http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh${VERSION} contrib\n" > /etc/apt/sources.list.d/cloudera.list
+
+# Prepare Cloudera repository
+COPY ${PATH_PREFIX}/conf/cloudera.pref /etc/apt/preferences.d/cloudera.pref
+RUN wget http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh/archive.key -O archive.key && \
+    sudo apt-key add archive.key && \
+    sudo apt-get update
+
+# Download necessary packages
+RUN apt-get install -y hadoop-conf-pseudo python-dev python-pip python-scipy && \
+    pip install requests colorama future tabulate pandas --upgrade
+
+# Initialize namenode
+RUN service hadoop-hdfs-namenode init
+
+# Add H2O user
+RUN useradd -ms /bin/bash h2o
```

Comment:
Why do you need h2o user?

Reply:
TBH it might be removed, it's just a habit to always create a user which "will actually do the job". Should I remove the H2O user?

Reply:
It is not related to this, but when I was doing the benchmarks docker, there was an issue with bower and npm when building h2o-3 under root. Maybe we can use the h2o user here as well, if you are not against.